### ISSUE TITLE
Conformance Group feedback wanted

### ISSUE THREAD

**Issue by GeorgeKerscher** [w3c/publ-a11y#169]
The Conformance Group is a new division in the principles section. The feedback so far is that it is a complex area and people do not understand WCAG references and levels. It should be made very simple and WCAG at the A level should be the minimum and level AA be the recommended. If there is third party certification, then this can also be included.

**Comment by GeorgeKerscher** [comment]
The example section was too complex and should be simplified. Here is the complex example that we intend to modify:

4.7.1Examples

The examples are provided first as a list of individual statements followed by a paragraph that concatinates statements that cover the whole conformance group. Either the list of individual statements or the paragraph style are appropriate, but do not provide both approaches. The conformance statement should be shown independently from the certifier and their associated information. Two examples are provided for the conformance statement, one shows a statement that meets recommended accessibility requirements and a second that shows a publication that is blow the recommended level.
EXAMPLE 16: Individual statements
• This publication meets accessibility recommendations and reports (EPUB Accessibility 1.1 and Web Accessibility Content Guidelines (WCAG) 2.1 Level AA.)
• This publication falls below the accessibility recommendations and reports (EPUB Accessibility 1.1 and Web Accessibility Content Guidelines (WCAG) 2.1 Level A.)
• This publication was certified on 2021-09-07.
• This publication is certified by Foo's Accessibility Testing (a third party.
• The certifier's credential is "A+ Accessibility Rating."
• For more information, refer to: Certifier's report.
EXAMPLE 17: Statements as a Paragraph
This publication meets accessibility recommendations and reports (EPUB Accessibility 1.1 and the Web Content Accessibility Guidelines (WCAG) 2.1 Level AA). The publication was certified on 2021-09-07 by Foo's Accessibility Testing (a third party) with the credential of "A+ Accessibility Rating." For more information refer to: Certifier's report.

**Comment by GeorgeKerscher** [comment]
I was thinking that the simple statements may be something like:
For a publication at the WCAG aa level withthird party certification:
* This publication is certified to meet the recommended accessibility standards.

or at the WCAG A level
* This publication is certified to meet the minimum accessibility standards.

or without third party certification at the WCAG AA level
* This publication meets the recommended accessibility requirements.

Or WCAG A without third party certification
* This publication meets the minimum accessibility requirements.

or for an optimized publication
* This publication is optimized to meet the accessibility needs of a specific group of users.

Then if the user wants to see all the metadata, either the list of individual statements or the paragraph style would be displayed.

**Comment by clapierre** [comment]
I like this language.

**Comment by mattgarrish** [comment]
> ... the recommended accessibility standards
> ... the recommended accessibility requirements

Was it intentional that you used two different expressions? It makes it sound like there are different criteria for non-certified content. I'd opt for "standards" for both.

> This publication is optimized to meet the accessibility needs of a specific group of users.

I don't know that this is helpful if you can't say what group. Also, absent a list of conformance IDs to match, which would allow for more specific language, how do you know the conformsTo statement relates to accessibility and not something else?

**Comment by mattgarrish** [comment]
> It may conform to a publisher's internal specification.

Right, this is what I'm worried about. Even if you insert the value of the standard, it may not mean anything, and it might not mean anything for accessibility.

Maybe there's some way of inferring the group. If the conformsTo statement is present, isn't recognized as an EPUB ID, and the primary access mode is auditory, then you could infer that the publication is optimized for auditory reading, for example. But what if the publication is optimized for dyslexic readers, for example? It feels like we'd need another piece of metadata to attach to the conformance statement to clarify the target audience.

**Comment by mattgarrish** [comment]
> and use Standards with a capital S

No need to capitalize since we're not giving a specific title.

**Comment by GeorgeKerscher** [comment]
Perhaps if it does not conformTo the values listed in 1.1 or 1.0 we should recommend that the publisher makes some clear statements in the accessibilitySummary.

**Comment by mattgarrish** [comment]
> we should recommend that the publisher makes some clear statements in the accessibilitySummary

This is part of the recommendations in the specification:

> If the identifier is not sufficient for a user to understand conformance (e.g., the guidelines are not publicly available), EPUB creators should provide additional information about they have optimized the content in the [accessibility summary](https://www.w3.org/TR/epub-a11y-11/#confreq-schema-accessibilitySummary).

Are optimizations likely to be distributed outside of specialized markets, though? If not, perhaps we only need to give general guidance in this area. For example:

> This publication is optimized to meet the accessibility needs of [insert user group here].

Where the distributor, knowing the content they deal in, can fill the user group in based on the identifiers they encounter.

**Comment by clapierre** [comment]
FYI: Publishers are putting in their own conformance statements.

For example this is what Macmillan Learning adds to their EPUBs along with another conformsTo statement to the original 1.0 IDPF conformance link.

`    <meta property="dcterms:conformsTo">Macmillan Learning Flat Accessible ePub Specification V5.0</meta>
`

**Comment by gautierchomel** [comment]
Closing as it has been addressed. See [Conformance section in actual UX guide draft](https://rawcdn.githack.com/w3c/publ-a11y/f7f9921cde19fd819683e9ea420482577f84088f/UX-Guide-Metadata/draft/principles/index.html#conformance-group)
